health-sdk/sdk/src/test/java/net/gini/android/health/sdk/paymentComponent/PaymentComponentViewTest.kt

```diff
@@ -0,0 +1,168 @@
+package net.gini.android.health.sdk.paymentComponent
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.text.SpannableString
+import android.text.style.ClickableSpan
+import android.widget.Button
+import android.widget.TextView
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import com.google.common.truth.Truth
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.test.runTest
+import net.gini.android.health.sdk.R
+import net.gini.android.health.sdk.paymentcomponent.PaymentComponent
+import net.gini.android.health.sdk.paymentcomponent.PaymentComponentView
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class PaymentComponentViewTest {
+    private var activity : Activity? = null
+    private var context: Context? = null
+    private var scenario: ActivityScenario<Activity>? = null
+    private lateinit var paymentComponent: PaymentComponent
+    private lateinit var paymentComponentListener: PaymentComponent.Listener
+
+    @Before
+    fun setUp() {
+        activity = Robolectric.buildActivity(Activity::class.java).create().get()
```

Comment:
[Suggestion] `activity` is not used and I checked quickly that it also works if we just call `Robolectric.buildActivity...` without assigning it to anything:
```suggestion
        Robolectric.buildActivity(Activity::class.java).create().get()
```

Maybe move it down to line 44 and leave comment that it's needed for the ActivityScenario because it's not obvious why we need it.